website/versioned_docs/version-2.0.0-alpha.63/docusaurus-core.md

```diff
@@ -298,6 +298,27 @@ const MyComponent = () => {
 };
 ```
 
+### `onRouteUpdate({location})`
+
+Set the page field to current location on the default tracker, such that subsequent hits on this page are attributed to this page.
+Recommended for Single-page Applications.
+
+Usage example:
+
+```jsx {2,5,6,7}
+import React from 'react';
+import {onRouteUpdate} from '@docusaurus/useGlobalData';
+
+const MyComponent = () => {
+  const location={
+                  pathname:'./example.html'
+                 };
```

Comment:
Fix the indentation problem 
`const location={ pathname:'./example.html' };`

Reply:
Thank you so much @saadpasta  for making this indentation fix point out.

Reply:
Thanks a lot. Fixed it ✌ [check here ](https://github.com/facebook/docusaurus/pull/3444/commits/55bc1ca017ece8fcb68c009653b674c818e194d8)